pipeline/devops-extension/tasks/libs/fusionPortalApi.ts

```diff
@@ -38,8 +38,8 @@ export default class PortalApi {
         await this.postAsync(endpoint);
     }
 
-    public async publishAppAsync(appKey : string) {
-        let endpoint = `${this.serverHost}/bundles/apps/${appKey}/publish`
+    public async publishAppAsync(appKey : string, versionId : string) {
+        let endpoint = `${this.serverHost}/api/apps/${appKey}/versions/${versionId}`
```

Comment:
I think we should have an endpoint where we can publish whatever the latest unpublished version is, instead of specifying the versionId. Not sure if we have the versionId in a deployment where we deploy and publish an app in the same pipeline

Reply:
> I think we should have an endpoint where we can publish whatever the latest unpublished version is, instead of specifying the versionId. Not sure if we have the versionId in a deployment where we deploy and publish an app in the same pipeline

I have created a new endpoint (adjusted in this file) allowing for publishing latest version (409 if already published).